it/mail/chrome/messenger/messenger.properties

```diff
@@ -569,38 +569,38 @@ confirmNavigateAwayFromQuickReplyTitle=Abbandonare la risposta attuale?
 confirmNavigateAwayFromQuickReply=I contenuti della tua risposta attuale andranno persi.
 
 # dataPane.js
-dataPaneShowMoreLink=more...
+dataPaneShowMoreLink=Mostra di più...
 
 #pbDeliciousService.js
 delicious.loginDialog.title=del.icio.us
-delicious.loginDialog.check.text=Utilizzare Gestione password per memorizzare questa password.
-delicious.loginDialog.text=Enter your username and password for del.icio.us.
+delicious.loginDialog.check.text=Utilizza Gestione password per memorizzare questa password.
```

Comment:
CORRECTIONS NEEDED.

Please replace "Gestione" with "Gestore" to improve the consistency.